ocaml/xapi/xapi_host.ml

```diff
@@ -1762,6 +1762,7 @@ let allocate_resources_for_vm ~__context ~self ~vm ~live =
   ()
 
 let set_iscsi_iqn ~__context ~host ~value =
+  if value = "" then raise Api_errors.(Server_error (invalid_value, ["value"; value]));
```

Comment:
maybe make the error description a bit more explicit that the iqn is wrong, or will that be obvious from the API call itself?

Reply:
Yes, I chose this as this seems to be the standard error we throw in these cases. I specified here the iqn param name, `"value"`, as that seems to be the convention for this error param.